packages/react/package.json

```diff
@@ -13,8 +13,8 @@
   "author": "Sam Mason de Caires <sam.jbmason@gmail.com>",
   "repository": "sammdec/nfte",
   "scripts": {
-    "build": "microbundle -f es,cjs --jsx React.createElement --define API_HOST='https://nfte.app' --css inline",
```

Comment:
Would you mind swithcing these back around

Reply:
Good catch! I swapped this out so I could run our Nextjs build in parallel on port `3000` while still being able to query the API endpoint :)